### PR TITLE
chore: added questions on cryptography in security-quiz.ts

### LINKED ISSUE #104: Adding Security questions to quiz site
### What general issue would you like to create?

[developerquiz.org](developerquiz.org) currently has over 600 quiz questions. We are looking to expand on the security  questions and we encourage other developers to add their quiz questions to the site.

You can find the complete list of security questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/security-quiz.ts

You can add your own questions to the top of that file.
**Please first check to make sure your question doesn't already exist in the file before creating a PR.**

Here is an example format for the questions. 

```
  {
    Question:
      "Which of the following type of Hackers protect civilians and organizations by finding and reporting vulnerabilities in the system?",
    Answer: "White Hat Hackers",
    Distractor1: "Grey Hat Hackers",
    Distractor2: "Black Hat Hackers",
    Distractor3: "Red Hat Hackers",
    Explanation:
      "White Hat Hackers work for organizations as Cybersecurity Engineers, Penetration Testers, etc.  where they perform tasks like ethically hacking the system to find and report vulnerabilities. ",
    Link: "https://www.freecodecamp.org/news/white-hat-black-hat-red-hat-hackers/",
  },
```
For the `link` field, please make sure to use a freeCodeCamp article, freeCodeCamp YouTube video or official documentation.
If you choose to reference a video, please make sure to include the timestamp for the topic covered.

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#). 

There are plenty of freeCodeCamp news articles to pull Security questions from:
https://www.freecodecamp.org/news/tag/web-security/
https://www.freecodecamp.org/news/tag/cybersecurity/
https://www.freecodecamp.org/news/tag/information-security/
https://www.freecodecamp.org/news/tag/security/

This issue will not be assigned to anyone and will remain open for multiple contributors. 
**Please do not assign yourself to this issue or close it.**

Happy contributing!

### PR DESCRIPTION
**Pull request by J-ell**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

### CODE REVIEW

**Comment by jdwilkin4** [comment]
I have removed the `Closes #104` from your PR.
Please make sure not to include that because we want this issue to stay open for multiple contributors.

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -8,6 +8,33 @@ const securityQuiz = [
     Explanation: "There is no attack model named CVA (Chosen vector attack).",
     Link: "https://en.wikipedia.org/wiki/Attack_model"
   },
+  {
+    Question: "Which of the following reasons is what makes HTTPS more secure than HTTP?",
+    Answer: "In HTTPS, all communications are encrypted", 
+    Distractor1: "HTTPS communications are in plain text",
+    Distractor2: "HTTPS communications are beautiful",
+    Distractor3: "HTTPS uses port 80 by default, therfore it is more secured",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:16]:
```suggestion
    Distractor3: "HTTPS uses port 80 by default, therefore it is more secure",
```

File: src/data/security-quiz.ts [hunk 2]
```diff
@@ -8,6 +8,33 @@ const securityQuiz = [
...
+    Explanation: "HTTPS ensures that all communications between a client(web browser) and a web server are completely encrypted.",
+    Link: "https://www.freecodecamp.org/news/http-vs-https/", 
+  },
+  {
+    Question: "What does GPG stands for?",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:21]:
```suggestion
    Question: "What does GPG stand for?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by J-ell** [comment]
> Congrats on another contribution to our repository. 🎉 We look forward to future contributions. 👍

Thank you for always making the experience awesome

**Pull request merged**